vault/testing.go

```diff
@@ -867,7 +875,7 @@ func (c *TestCluster) UnsealCoresWithError() error {
 
 	// Ensure cluster connection info is populated.
 	// Other cores should not come up as leaders.
-	for i := 1; i < numCores; i++ {
+	for i := 1; i < len(c.Cores); i++ {
```

Comment:
Should we also check that Core 0 is in fact the leader?

Reply:
TestWaitActiveWithError does so.  Leader == Active Node.